workspaces/liferay-ticket-workspace/client-extensions/liferay-ticket-custom-element/src/main.tsx

```diff
@@ -3,26 +3,103 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
+import {ClayVerticalNav} from '@clayui/nav';
+import {useState} from 'react';
 import {createRoot} from 'react-dom/client';
 import {QueryClient, QueryClientProvider} from 'react-query';
+import {HashRouter, Route, Routes} from 'react-router-dom';
 
 import TicketApp from './pages/TicketApp';
+import TicketsByStatusDashboard from './pages/TicketsByStatusDashboard';
+import {Liferay} from './services/liferay';
 
+import './styles/Main.css';
+export const SPRITEMAP =
+	(Liferay as any).ThemeDisplay.getPathThemeImages() + '/clay/icons.svg';
+
+const DEFAULT_ROUTE: string = '#dashboard';
 const queryClient = new QueryClient();
-class TicketWebComponent extends HTMLElement {
+
+const Main: React.FC = () => {
+	const [activeItem, setActiveItem] = useState('1');
+
+	if (window.location.href.indexOf('#') === -1) {
+		window.location.href = window.location.href + DEFAULT_ROUTE;
```

Comment:
We don't need to do this. If the default route is `#/dashboard` and we can't remove the slash, that's fine. We should just make all links/routes follow the same pattern. IE: below it should be: `href: '#/dashboard'`.